modules/lobbies/scripts/create.ts

```diff
@@ -33,6 +36,15 @@ export async function run(
 	ctx: ScriptContext,
 	req: Request,
 ): Promise<Response> {
+	await ctx.modules.captcha.guard({
+		key: "lobbies.create",
+		period: 15,
```

Comment:
period & requests should be configurable in config.ts and have defaults in the default config